### PR TITLE
Add Ukrainian translation for typing-game

### PR DESCRIPTION
**Pull request by therrance**
issue #489

### CODE REVIEW

**Comment by jlooper** [comment]
Is there a way for our Ukrainian translators to double-check the translations? Cc @Art-auto

**Review by jlooper — CHANGES_REQUESTED** [review]
We think that the code to use is .uk for Ukrainian

**Comment by therrance** [comment]
@jlooper, of course, thank you for review

File: 4-typing-game/translations/README.uk.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Цей урок передбачає, що ви знайомі з такими поняттями:
+
+- Створення елементів керування введенням тексту та кнопок
+- CSS і налаштування стилів за допомогою класів
+- Основи JavaScript
+  - Створення масиву
+  - Створення випадкового числа
+  - Отримання поточного часу 
+
+## Уроки
```
Comment (Art-auto) [inline 4-typing-game/translations/README.uk.md:24]:
```suggestion
## Урок
```

**Comment by Art-auto** [comment]
Please check the paths back to the root lessons, you may need ../

File: 4-typing-game/translations/README.uk.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Програмування на основі подій – створимо гру для введення тексту
+
+## Вступ
+
+Введення тексту — одна з найбільш недооцінених навичок розробника. Здатність швидко передавати думки з голови до редактора дозволяє вільно протікати творчості. Один з найкращих способів навчитися - грати в гру!
+
+> Отже, давайте створимо гру для набору тексту!
+
+Ви збираєтеся використовувати навички JavaScript, HTML і CSS, які ви вже напрацювали, щоб створити гру набору тексту. Гра представить гравцеві випадкову цитату (ми використовуємо цитати [Шерлока Холмса](https://uk.wikipedia.org/wiki/%D0%A8%D0%B5%D1%80%D0%BB%D0%BE%D0%BA_%D0%A5%D0%BE%D0%BB%D0%BC%D1%81) ) і час, скільки часу потрібно гравцеві, щоб точно ввести його. Ви збираєтеся використовувати навички JavaScript, HTML і CSS, які ви вже напрацювали, щоб створити гру набору тексту.
```
Comment (Art-auto) [inline 4-typing-game/translations/README.uk.md:9]:
```suggestion
Ви збираєтеся використовувати навички JavaScript, HTML і CSS, які ви вже напрацювали, щоб створити гру набору тексту. Гра представить гравцеві випадкову цитату (ми використовуємо цитати [Шерлока Холмса](https://en.wikipedia.org/wiki/Sherlock_Holmes) ) і час, скільки часу потрібно гравцеві, щоб точно ввести його. Ви збираєтеся використовувати навички JavaScript, HTML і CSS, які ви вже напрацювали, щоб створити гру набору тексту.
```

**Comment by Art-auto** [comment]
The translation itself looks fine

**Comment by therrance** [comment]
Thank you, @Art-auto the paths was fixed.

**Pull request merged**